jena-parent/pom.xml

```diff
@@ -258,37 +257,92 @@
         <version>${ver.lucene}</version>
       </dependency>
 
-      <!-- Solr client -->
-      <!-- Exclusion of slf4j: Necessary so as to pick the version we want. 
-           solrj->zookeeper has a dependency on slf4j -->
+      <!-- For jena-spatial -->
+      <dependency>
+        <groupId>org.apache.lucene</groupId>
+        <artifactId>lucene-spatial</artifactId>
+        <version>${ver.lucene}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.apache.lucene</groupId>
+        <artifactId>lucene-spatial-extras</artifactId>
+        <version>${ver.lucene}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.locationtech.spatial4j</groupId>
+        <artifactId>spatial4j</artifactId>
+        <version>${ver.spatial4j}</version>
+      </dependency>
 
+      <!-- ES dependencies-->
       <dependency>
-        <artifactId>solr-solrj</artifactId>
-        <groupId>org.apache.solr</groupId>
-        <version>${ver.solr}</version>
+        <groupId>org.elasticsearch</groupId>
+        <artifactId>elasticsearch</artifactId>
+        <version>${ver.elasticsearch}</version>
         <exclusions>
           <exclusion>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
           </exclusion>
           <exclusion>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-jdk14</artifactId>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-core</artifactId>
           </exclusion>
         </exclusions>
+
       </dependency>
 
-      <!-- For jena-spatial -->
+      <dependency>
+        <groupId>org.elasticsearch.client</groupId>
+        <artifactId>transport</artifactId>
+        <version>${ver.elasticsearch}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-core</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+
+
       <dependency>
         <groupId>org.apache.lucene</groupId>
-        <artifactId>lucene-spatial</artifactId>
+        <artifactId>lucene-test-framework</artifactId>
```

Comment:
Is the dependency on lucene-test-framework necessary? I commented it out in the POM and the build and tests ran just fine.